README.md

```diff
@@ -38,13 +38,14 @@ Remember these guidelines:
 
 The shapes have been designed with the specific intention of being usable by everyone including those that may have a color vision deficiency. For this reason, the following constraints should apply:
 
-- Platform teams should always have square corners 
+- Platform teams should always have square corners
 - Stream-aligned teams should always be horizontally aligned with rounded corners
 - Enabling teams should always be vertically aligned with rounded corners
 - Complicated subsystem teams should always be an octagon
+- Team should always be horizontally aligned with rounded corners and dotted border
```

Comment:
An Undefined Team Type team should always be horizontally aligned with rounded corners and dotted border